docs/en/stack/data-frames/limitations.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[dataframe-limitations]]
+== {dataframe-cap} limitations
+
+beta[]
+
+The following limitations and known problems apply to the 7.2 release of 
+the Elastic {dataframe} feature:
+
+[float]
+[[df-wizard-limitations]]
+=== {dataframe-cap} wizard limitation
```

Comment:
Data frames does not (yet) support fields containing arrays - in the UI or the API.

If you try to create one, the UI will fail to show the source index table (which is not the main point of this limitation)

Reply:
I was able to create them (or so I thought) while playing with the eCommerce data. Are they unsupported nonetheless?